src/DropDown/DropDown.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { DropDown } from './DropDown';
+
+export default {
+  title: 'Components/DropDown',
+  argTypes: { onChange: { action: 'changed' } },
+  component: DropDown,
+} as Meta;
+
+const itemsWithImages = [
+  {
+    label: 'Item 1',
+    value: 'item1',
+    thumbnail:
+      'https://is2-ssl.mzstatic.com/image/thumb/Purple69/v4/6f/b1/4a/6fb14a9a-d356-0d33-69c9-2f39ed7f0ecd/source/256x256bb.jpg',
+  },
+  {
+    label: 'Item 2',
+    value: 'item2',
+    thumbnail: 'https://i1.pngguru.com/preview/350/922/403/classic-car-icon-256-png-icon.jpg',
+  },
+  {
+    label: 'Item 3',
+    value: 'item3',
+    thumbnail: 'https://i1.pngguru.com/preview/350/922/403/classic-car-icon-256-png-icon.jpg',
+  },
+];
+
+const items = [
+  {
+    label: 'Item 1',
+    value: 'item1',
+  },
+  {
+    label: 'Item 2',
+    value: 'item2',
+  },
+  {
+    label: 'Item 3',
+    value: 'item3',
+  },
+];
+
+export const WithImages: Story = args => (
+  <DropDown items={itemsWithImages} onChange={args.onChange} initialSelectedItem={itemsWithImages[1]} />
```

Comment:
Can you wrap the dropdowns in a div that is a like 200px high, so it is easy to see that the list is open?

Reply:
Doing this seems to make no difference? On mine I can see the dropdown and if I click it it opens?

Reply:
In the docs section it helps see what is happening.
<img width="1479" alt="Screenshot 2020-10-09 at 10 53 06" src="https://user-images.githubusercontent.com/30238668/95569595-b318ac00-0a1d-11eb-8b45-8679da4bc412.png">